include/rdma/fabric.h

```diff
@@ -121,6 +121,7 @@ typedef struct fid *fid_t;
 #define FI_TRANSMIT_COMPLETE	(1ULL << 27)
 #define FI_DELIVERY_COMPLETE	(1ULL << 28)
 
+#define FI_SELECTIVE_COMPLETION	(1ULL << 55)
```

Comment:
Oh, I think I see it now.  The `59-go down` space is reused multiple times for different classes.  That was... not intuitive to me.